binding/dotnet/PvSpeaker/PvSpeaker.csproj

```diff
@@ -49,6 +49,16 @@
             <Link>lib\windows\amd64\libpv_speaker.dll</Link>
             <Visible>false</Visible>
         </Content>
+        <Content Include="..\..\..\lib\windows\arm64\libpv_speaker.dll">
+            <PackagePath>
+                build/netcoreapp3.0/lib/windows/arm64/libpv_speaker.dll;
+                build/net6.0/lib/windows/arm64/libpv_speaker.dll;
+                build/net8.0/lib/windows/arm64/libpv_speaker.dll;
```

Comment:
you may need to check if arm64 Windows can even run on .NET core 3.0 - I believe it may not have been supported until .NET 6.0